src/Controller/EnMarche/SearchController.php

```diff
@@ -76,11 +75,27 @@ public function searchCommitteesAction(Request $request)
      * @Route("/projets-citoyens", name="app_search_citizen_projects")
      * @Method("GET")
      */
-    public function searchAction(Request $request): Response
+    public function searchCitizenProjectsAction(Request $request)
     {
         $this->disableInProduction();
 
-        return new Response();
+        $request->query->set(SearchParametersFilter::PARAMETER_TYPE, SearchParametersFilter::TYPE_CITIZEN_PROJECTS);
+
+        $search = $this->getSearch($request);
+
+        try {
+            $results = $this->get('app.search.search_results_providers_manager')->find($search);
+        } catch (GeocodingException $exception) {
+            $errors[] = $this->get('translator')->trans('search.geocoding.exception');
+        }
+
+        return $this->render('search/search_citizen_projects.html.twig', [
+            'search_max_results' => $this->getParameter('search_max_results'),
```

Comment:
:/, and here's another reason why this should be part of the search.

Reply:
You are right